conformance/tests/admin-network-policy-core-ingress-tcp-rules.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: policy.networking.k8s.io/v1alpha1
```

Comment:
I might rename this as `admin-network-policy-core-ingress-tcp-rules_base.yaml` since we actually edit the yaml in memory in the test itself (swapping around rules)

Reply:
ack, let me rename the files to have the `_base` in names.